scripts/check-license.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ret=0
+
+for file in $(find . -type f -iname '*.go' ! -path './vendor/*' ! -path './proto/*.pb.*'); do
+	if ! head -n3 "${file}" | grep -Eq "Copyright © 2017 thingful"; then
```

Comment:
Do you think it would be better to use the current year dynamically rather than hard code it to 2017?
We would then have to change update the copyright every January but I think I'm fine with that - what do you think?